plugins/CMakeLists.txt

```diff
@@ -165,6 +166,10 @@ set (DARTplugins
   GravityCompensationPlugin
 )
 
+set(GAZEBO_PLUGIN_INSTALL_DIR
```

Comment:
Interesting, I don't see this variable used anywhere, and the commit seems to be from 7 years ago

Reply:
ouch, thanks removed here 580b504